scripts/dqlite/scripts/repl/repl.sh

```diff
@@ -3,5 +3,25 @@
 set -e
 
 MACHINE=${MACHINE:-0}
+DB_NAME=${DB_NAME:-controller}
 
-juju ssh -m controller ${MACHINE} 'sudo rlwrap -H /root/.dqlite_repl.history socat - /var/lib/juju/dqlite/juju.sock'
+echo "-------------------------------------------------------------------------"
+echo ""
+echo "DQLITE REPL Mode:"
+echo ""
+echo "Once logged in, run the following command:"
+echo ""
+echo " > dqlite -s \$(sudo cat /var/lib/juju/dqlite/info.yaml | grep \"Address: \" | cut -d\" \" -f2) ${DB_NAME}"
+echo ""
+echo "-------------------------------------------------------------------------"
+echo ""
+echo "                             WARNING!"
+echo ""
+echo "You're attached to the live database. There currently is no audit trail"
```

Comment:
Should be "This will attach you to the live database".
"running any commands" at the end of 2 sentences doesn't flow nicely. I'd drop the first instance of it.

If there's any way to compress this, that would be nice. On my terminal, MOTD pushes it up through the top.